std_msgs/Header header
string[] class_name
float64[] diagonal_length
float64[] center_x
float64[] center_y
int32[] ids 